userdocs/profiles.dev/docs/author-docs/profile-structure.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 1
+---
+
+# How a profile is structured
+
+A profile is defined in a single file which **must** be named `profile.yaml`.
+
+It will be assumed that everything contained within the same directory as a `profile.yaml`
+is related to that same profile.
+
+A repository can contain multiple profiles when they are written in separate directories.
+For example, the following structure shows a repo with three distinct profiles which
+can be installed independently of each other:
+
+```bash
+org-profiles-repo/
+├── logging-profile
+│   └── profile.yaml
+├── observability-profile
+│   └── profile.yaml
+└── our-awesome-apps
+    └── profile.yaml
+```
+
+A repository can also contain just a single profile, with the `profile.yaml`
+defined at the top level:
+
```

Comment:
when using folders the folder name must == profile.yaml metadata.name

Reply:
ah ty 👍